agent/lib/kontena/workers/stats_worker.rb

```diff
@@ -34,9 +34,9 @@ def on_node_info(topic, info)
     end
 
     def start
-      info 'waiting for cadvisor'
+      debug 'waiting for cadvisor'
       sleep 1 until cadvisor_running?
-      info 'cadvisor is running, starting stats loop'
+      debug 'cadvisor is running, starting stats loop'
```

Comment:
Imho this is valuable information.. and it is only logged when agent starts (does not spam logs).